examples/question_answering/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "question_answering"
+version = "0.1.0"
+description = "Example Kolena integration for question answering"
+authors = ["Kolena Engineering <eng@kolena.io>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.10"
```

Comment:
Is python 3.9 the only version that works? Ideally our examples work with a wide range of python versions.